http4s-blaze-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeClientConfig.scala

```diff
@@ -0,0 +1,30 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.ssl.SslContextConfig
+import org.http4s.BuildInfo
+import org.http4s.client.blaze.ParserMode
+import org.http4s.client.defaults
+import org.http4s.headers.{`User-Agent`, AgentComment, AgentProduct}
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sBlazeClientConfig(
+  responseHeaderTimeout: Duration = Duration.Inf,
+  idleTimeout: FiniteDuration = Duration(1, TimeUnit.MINUTES),
+  requestTimeout: FiniteDuration = defaults.RequestTimeout,
+  connectTimeout: FiniteDuration = defaults.ConnectTimeout,
+  userAgent: `User-Agent` = `User-Agent`(AgentProduct("http4s-blaze-client", Some(BuildInfo.version)), List(AgentComment("Server"))),
+  maxTotalConnections: Int = 10,
+  maxWaitQueueLimit: Int = 256,
+  maxConnectionsPerRequestkey: Int = 256,
+  sslContext: Option[SslContextConfig] = None,
```

Comment:
maybe it would be for the best, if `SslContextConfig` wasn't part of `Http4sBlazeClientConfig`, and was provided at the required places separately (either the `SslContextConfig` would be provided, or already a `SslContext` itself)

Reply:
As discussed personally let's keep it this way and extend if we ever need it.